djangorestframework_camel_case/util.py

```diff
@@ -17,9 +17,10 @@ def camelize(data):
             new_dict[new_key] = camelize(value)
         return new_dict
     if isinstance(data, (list, tuple)):
+        new_sequence = []
         for i in range(len(data)):
-            data[i] = camelize(data[i])
-        return data
+            new_sequence.append(camelize(data[i]))
```

Comment:
I tried to mimic the way it looks in dict `if` branch, but maybe You're right. I have changed it to list comprehension.